lxqthtmldelegate.cpp

```diff
@@ -98,13 +98,13 @@ void HtmlDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
     const QRect clip(0, 0, options.rect.width() - iconRect.width() - 8, options.rect.height());
     painter->setClipRect(clip);
 
-    // set text color to red for selected item
+    // set text colors
     QAbstractTextDocumentLayout::PaintContext ctx;
-    if (option.state & QStyle::State_Selected)
-    {
-        QPalette::ColorGroup colorGroup = (option.state & QStyle::State_Active) ? QPalette::Active : QPalette::Inactive;
+    QPalette::ColorGroup colorGroup = (option.state & QStyle::State_Active) ? QPalette::Active : QPalette::Inactive;
```

Comment:
Shouldn't everywhere here be used `options` after `initStyleOption()`?

Reply:
We want the palette so, at least, it makes no difference.

BTW, the whole code was copy/pasted from an answer in stackoverflow -- half correct half not -- and apparently, no one had time to clean it up.